### DISCUSSION TITLE
음파 통신 속도 2배를 위한 사전 스터디

### DISCUSSION THREAD

**Discussion by dlwlstks96** [euphony-io/check-in-listener#13] [General]
![마스터코스_미션](https://user-images.githubusercontent.com/72801465/133888933-c877522c-06c2-41c6-9026-9687b654e374.JPG)
@YoungSeokHong 
1. 사전에 멘토님께서 작성해주셨던 내용입니다. 
FFT에 관한 좋은 영상을 찾은 것 같아 공유 드립니다!
[푸리에 변환(FFT)](https://www.youtube.com/watch?v=spUNpyF58BY)

2. 또한 리스너에 해당하는 자바 코드만 분석하다 위 이미지에 나와 있는
BASE 인터페이스를 cpp 폴더 내에서 찾은듯 하여 말씀 드립니다!
`euphony - src - main - cpp - core - Base.h`

아직 저도 완전히 이해한 것이 아니라 계속해서 공부해보겠습니다!

**Comment by designe** [comment]
@dlwlstks96 좋네요! 진산님 이 부분은 음파 발생 쪽과 한글패치 연구하시는 분들 모두에게 도움이 될 것 같아서
euphony repo의 discussion에 써주셔도 좋을 것 같네요 :)

**Reply by dlwlstks96** [reply]
네 알겠습니다!!
